vendor/metal-cpp/Metal/MTLEvent.hpp

```diff
@@ -62,6 +62,7 @@ class SharedEvent : public NS::Referencing<SharedEvent, Event>
 
     uint64_t                 signaledValue() const;
     void                     setSignaledValue(uint64_t signaledValue);
+    bool                     waitUntilSignaledValue(uint64_t signaledValue, uint64_t timeoutMS);
```

Comment:
For our purposes they are all the same.  I have to have it included here because the SwiftPM build can't check out, un .tar.gz and build it.  Well, actually it _can_ do those things but not also link them.

The patch is just for documentation -- in the readme I said where the vendor code is from and what patch I applied.  This is just keeping it all up to date.